KD_Lib/original/train.py

```diff
@@ -0,0 +1,63 @@
+import torch.nn.functional as F
+import matplotlib.pyplot as plt
+
+
+def train_teacher(model, train_loader, optimizer, epochs, device):
+
+    model.train()
+    loss_arr = []
+
+    print('Training teacher - \n')
+
+    for e in range(epochs):
+        epoch_loss = 0
+        for (data, label) in train_loader:
+            data, label = data.to(device), label.to(device)
```

Comment:
Also device isn't defined anywhere. Please make these changes. The build should pass now.